### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#245] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152314996-f28ba57a-18cf-4d4c-b073-c83b08ae16b8.png)

**Comment by GattoBrizzolato** [comment]
![es5](https://user-images.githubusercontent.com/98970164/172479727-346581bf-95b9-4a18-9cca-09fa66fe941f.jpeg)

**Comment by Elia-Belli** [comment]
L'insieme dato rappresenta una funzione f: $\mathbb{Z}$ -> $\mathbb{Z}$ che opera secondo la legge y = 4-3x (scritta in forma esplicita): 

- Le operazioni di somma e prodotto tra numeri interi negativi restituiscono sempre numeri interi negativi.

Allora f è una funzione.
